src/Channel.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * Created by kenshoo-linux on 14/07/16.
+ */
+/**
+ * Created by ken-linux2 on 12/07/16.
+ */
+public class Channel {
+
+    private String name;
+    private String channel_campaign_id;
```

Comment:
BTW, no need to call it channelCampaignId since we are in the Channel class, so just call it campaignId, same goes to the other one